### ISSUE TITLE
[BUG] CLI `call --running` flag is interpreted as `--help`

### ISSUE THREAD

**Issue by jost-s** [holochain/holochain#1423]
**Describe the bug**
Issuing a call to a sandbox conductor shows the help for the `call` subcommand when using flag `--running`.
Command output is:
```sh
$ hc s call --running 30000 list-cells
hc-sandbox-call 0.0.33
Make a call to a conductor's admin interface

USAGE:
    hc sandbox call [FLAGS] [OPTIONS] [indices]... <SUBCOMMAND>

FLAGS:
    -a, --all        Run all the existing conductor sandboxes
    -h, --help       Prints help information
    -l, --last       Run the last created conductor sandbox
    -V, --version    Prints version information

OPTIONS:
    -e, --existing-paths <existing-paths>...    Paths to existing sandbox directories. For example `hc run
                                                -e=/tmp/kAOXQlilEtJKlTM_W403b,/tmp/kddsajkaasiIII_sJ`
    -r, --running <running>...                  Ports to running conductor admin interfaces. If this is empty existing
                                                sandboxes will be used. Cannot be combined with existing sandboxes

ARGS:
    <indices>...    Run a selection of existing conductor sandboxes. Existing sandboxes are visible via `hc list`.
                    Use the index to choose which sandboxes to use. For example `hc run 1 3 5` or `hc run 1`

SUBCOMMANDS:
    add-admin-ws          Calls AdminRequest::AddAdminInterfaces and adds another admin interface
    add-agents            Calls AdminRequest::AddAgentInfo. _Unimplemented_
    add-app-ws            Calls AdminRequest::AttachAppInterface and adds another app interface
    disable-app           Calls AdminRequest::DisableApp and disables the installed app
    dump-state            Calls AdminRequest::DumpState and dumps the current cell's state. TODO: Add pretty print.
                          TODO: Default to dumping all cell state
    enable-app            Calls AdminRequest::EnableApp and activates the installed app
    help                  Prints this message or the help of the given subcommand(s)
    install-app           Calls AdminRequest::InstallApp and installs a new app
    install-app-bundle    Calls AdminRequest::InstallAppBundle and installs a new app
    list-active-apps      Calls AdminRequest::ListActiveApps
    list-agents           Calls AdminRequest::RequestAgentInfo and pretty prints the agent info on this conductor
    list-app-ws           Calls AdminRequest::ListAppInterfaces
    list-apps             Calls AdminRequest::ListApps
    list-cells            Calls AdminRequest::ListCellIds
    list-dnas             Calls AdminRequest::ListDnas
    new-agent             Calls AdminRequest::GenerateAgentPubKey
    register-dna          Calls AdminRequest::RegisterDna and registers a Dna. You can only use a path or a hash not
                          both
    uninstall-app         Calls AdminRequest::UninstallApp
```

**Expected behavior**
The conductor call should be executed, using the admin interface port specified by `--running`.

**System information:**
 - OS: macOS 12.4
 - Holochain and HDK Version:
 ```bash
$ hn-introspect
List of applications and their version information

v0_0_139
- hc-0.0.37: https://github.com/holochain/holochain/tree/holochain-0.0.139
- holochain-0.0.139: https://github.com/holochain/holochain/tree/holochain-0.0.139
- kitsune-p2p-tx2-proxy-0.0.24: https://github.com/holochain/holochain/tree/holochain-0.0.139
- lair-keystore-0.0.10: https://github.com/holochain/lair/tree/v0.0.10

- rustc: rustc 1.60.0 (7737e0b5c 2022-04-04)
- cargo fmt: rustfmt 1.4.38-stable (7737e0b5 2022-04-04)
- cargo clippy: clippy 0.1.60 (7737e0b5 2022-04-04)
```

```bash
$ holochain --build-info | jq
{
  "git_info": null,
  "cargo_pkg_version": "0.0.139",
  "hdk_version_req": "0.0.132",
  "timestamp": "2022-05-16T09:36:22.604402Z",
  "hostname": "Mac-1652692418058.local",
  "host": "x86_64-apple-darwin",
  "target": "x86_64-apple-darwin",
  "rustc_version": "rustc 1.60.0 (7737e0b5c 2022-04-04)",
  "rustflags": "",
  "profile": "release"
}
```

**Steps to reproduce**
* enter a current holonix shell
* execute `hc s call --running 30000 list-cells` (no prior sandbox setup required)

**Comment by ThetaSinner** [comment]
This will be fixed by #2125

**Comment by abe-njama** [comment]
@jost-s This issue should now be resolved courtesy of the fix that was pushed in #2125. Can we proceed to close this issue?

**Comment by jost-s** [comment]
That branch has only been merged into another branch. It'll be fixed by #2100

**Comment by abe-njama** [comment]
Thanks @jost-s for pointing that out.

**Comment by ThetaSinner** [comment]
This final PR for this merged in #2100 and will be part of `0.3.0`. It was also backported to the 0.2.x series on #2362 which should release with `0.2.1`.
